main.js

```diff
@@ -22,15 +22,16 @@ function createWindow(on_load) {
     newWindow.loadURL('file://' + __dirname + '/index.html');
 }
 
-function newDocument() {
+function newProject() {
     createWindow(function(win) {
-        win.webContents.send('new-document');
+        win.webContents.send('new-project');
     });
 }
 
-function openDocument(path) {
+function openProject(path) {
     createWindow(function(win) {
-        win.webContents.send('open-document', path);
+        path = path.replace('app.tbinfo','');
```

Comment:
Actually, it might make more sense to have this code in the `dialog.showOpenDialog` callback, since this function is called 'openProject', and this is dealing with what the user selected in the open dialog.